Shared/Samples/Browse OGC API feature service/BrowseOGCAPIFeatureServiceView.swift

```diff
@@ -0,0 +1,242 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import ArcGISToolkit
+import SwiftUI
+
+struct BrowseOGCAPIFeatureServiceView: View {
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    /// Ensures that the alert is only shown once.
+    @State private var presentAlert = true
```

Comment:
```suggestion
    @State private var alertIsPresented = true
```
This boolean name should read as an assertion.